content/02.delete-me.md

```diff
@@ -1,15 +1,146 @@
-## Manubot Rootstock Information
+## Introduction
 
-**Note: Manubot instances should delete this file.**
+This manuscript is a template/boilerplate (aka "rootstock") for [Manubot](https://manubot.org/ "Manubot"), a tool for writing scholarly manuscripts.
+Use this template as a starting point for your manuscript by following the directions on the [repository homepage](https://github.com/greenelab/manubot-rootstock "Manubot Rootstock").
 
-The Manubot is a system for automating scholarly publishing.
-Content is written in [Pandoc Markdown](http://pandoc.org/MANUAL.html#pandocs-markdown) source files.
-See [`USAGE.md`](https://github.com/greenelab/manubot-rootstock/blob/master/USAGE.md) for more information on how to use the Manubot.
 
-The Manubot project began with the [Deep Review](https://github.com/greenelab/deep-review), where it was used to compose a highly-collaborative review article [@doi:10.1098/rsif.2017.0387].
-Another example manuscript that was created with Manubot is:
 
-+ The Sci-Hub Coverage Study ([GitHub](https://github.com/greenelab/scihub-manuscript), [HTML manuscript](https://greenelab.github.io/scihub-manuscript/)) [@doi:10.7554/eLife.32822]
+## Examples
 
-If you notice a problem with Manubot, it's best to submit an upstream fix to the appropriate repository:
-[`greenelab/manubot-rootstock`](https://github.com/greenelab/manubot-rootstock) for the git repository stub or [`greenelab/manubot`](https://github.com/greenelab/manubot) for the Python package.
+Below is a full list of formatting elements for `.html` and `.pdf` exports that are supported by the Manubot-provided themes. 
+Compare the input Markdown (`.md` files in the `/content` directory) to the output you see below.
+
+# Heading 1
```

Comment:
Should we add a note that this heading is reserved for the title?

Reply:
> Is there anything hard-coded into pandoc or manubot that assumes the existence of one and only one

No. But since authors don't put the title in the manuscript themselves, they may think that h1 should be used for their first level of headings... e.g. `Abstract`. So as long as we note that the title is h1 and sections are h2, that is sufficient. In rare cases someone may want to insert another h1 heading, although more often then not this is probably a mistake.

Reply:
> In rare cases someone may want to insert another h1 heading

I agree it is desirable in rare cases.  One of them would be creating a single document that includes the main text and supplement, where the supplement has a h1 header again.

Saying that h1 is for the title and h2 is for the sections sounds good.